packages/cli/src/commands/job.ts

```diff
@@ -0,0 +1,75 @@
+import ora from 'ora';
+import { runJob, getJobById, getJobByPipeline, getJobs, getLogById } from '../service/job';
+
+const PipelineStatus: any = {
+  0: 'creating',
+  1: 'running',
+  2: 'success',
+  3: 'fail'
+};
```

Comment:
use `enum` instead?

Reply:
enum is kind of map from string to number or string to string. This is from number to string~

Reply:
Why not use array:(
[ 'creating', 'running', 'success', 'fail' ] looks feasible:(

Reply:
> Why not use array:(
> [ 'creating', 'running', 'success', 'fail' ] looks feasible:(

Yes, I will modify